gufe/setup/network_planning/atom_mapping_based/atom_mapping_scorer.py

```diff
@@ -0,0 +1,43 @@
+# This code is part of kartograf and is licensed under the MIT license.
+# For details, see https://github.com/OpenFreeEnergy/gufe
+
+import abc
+from ....tokenization import GufeTokenizable
+
+from .atom_mapping import AtomMapping
+
+
+class AtomMappingScorer(GufeTokenizable):
```

Comment:
This is in line with what we discussed, so this feature I think it ok to include.